chore(migration-guide): add migration guide for DateTime and RelativeDateTime (#1505)
contentful/forma-36: MIGRATION.md

/>;
```

## DateTime

> The API of the DateTime component hasn't changed. The `format` property is the only one that was updated: from `"FULL"`, `"DATE_ONLY"`, `"TIME_ONLY"`, `"WEEKDAY_DATE"` to `"full"`, `"day"`, `"time"`, `"weekday"`.

Reviewer: ```suggestion
The API of the DateTime component has not changed. The `format` property is the only one that was updated: from `"FULL"`, `"DATE_ONLY"`, `"TIME_ONLY"`, `"WEEKDAY_DATE"` to `"full"`, `"day"`, `"time"`, `"weekday"`.
```

> The API of the DateTime component hasn't changed. The only property that has some updates, is `format` property. From `"FULL"`, `"DATE_ONLY"`, `"TIME_ONLY"`, `"WEEKDAY_DATE"` to `"full"`, `"day"`, `"time"`, `"weekday"`.

Reviewer: ```suggestion
The API of the DateTime component hasn't changed. The `format` property is the only one that was updated: from `"FULL"`, `"DATE_ONLY"`, `"TIME_ONLY"`, `"WEEKDAY_DATE"` to `"full"`, `"day"`, `"time"`, `"weekday"`.
```

```tsx static=true
import { DateTime } from '@contentful/f36-datetime';

<DateTime date="2020-08-17T15:45:00" />
<DateTime date="2020-08-17T15:45:00" format="day" />
<DateTime date="2020-08-17T15:45:00" format="weekday" />
<DateTime date="2020-08-17T15:45:00" format="time" />
```

### How to migrate your DateTime components

> Update the `format` value and import. It changes from this:

Reviewer: For consistency throughout the documentation should we phrase this as follows?

To migrate your DateTime component to v4, you must update the `format` value and import. It changes from this:

> Update `format` value and import. It should change form this:

Reviewer: ```suggestion
Update the `format` value and import. It changes from this:
```

```tsx static=true
import { DateTime } from '@contentful/forma-36-react-components';

<DateTime date="2020-08-17T15:45:00" format="WEEKDAY_DATE" />;
```

to this:

```tsx static=true
import { DateTime } from '@contentful/f36-components';

<DateTime date="2020-08-17T15:45:00" format="weekday" />;
```

## RelativeDateTime

> The API of the RelativeDateComponent was not changed.

Reviewer: ```suggestion
The API of the RelativeDateComponent has not changed.
```

### How to migrate your RelativeDateTime components

> You just need to update import. From this:

Reviewer: For consistency throughout the guide, should we phrase this as follows?

To migrate RelativeDateComponent to v4, you must update the import. From this:
